.gitignore

```diff
@@ -0,0 +1,15 @@
+# OS generated files #
+######################
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
+
+node_modules/
```

Comment:
Only node_modules i sass-cache should stay here, rest should go to your global .gitingnore file.